### ISSUE TITLE
[Enhancement] Enable Pyth price feeding for SOLANA Token Economy

### ISSUE THREAD

**Issue by cloud8little** [ExocoreNetwork/price-feeder#4]
Most SOLANA tokens are feeded price by pyth, so if we are going to support SOLANA client chain, we would introduce pyth to feed the token price.

**Comment by magj2006** [comment]
Yes, we should integrate Pyth’s price feed, because it supports most asset price feeds on non-EVM chains. We can get more information from its official website (https://www.pyth.network/).

**Comment by leonz789** [comment]
Why do we need to use pyth ?
